lib/metadata_fields.rb

```diff
@@ -0,0 +1,215 @@
+module MetadataFields
+	extend ActiveSupport::Concern
+	extend self
+
+	NAMESPACE_ALL = 'all'
+	MULTIPLE_ALL  = 'all'
+	DEFAULT_NAMESPACE = NAMESPACE_ALL
+	DEFAULT_MULTIPLE  = MULTIPLE_ALL
+	METADATA_FIELDS = YAML.load_file(File.join(Rails.root, "config", "metadata_fields.yml"))["terms"]["vocabulary"].deep_symbolize_keys
```

Comment:
could use a constant for `YAML` file contents. 
e.g., `CONFIG_DATA = YAML.load_file(File.join(Rails.root, "config", "metadata_fields.yml"))` then use the constant in the other initializations:
`METADATA_FIELDS = CONFIG_DATA['terms']['vocabulary'].deep_symbolize_keys` 
`DISPLAY_CONFIG = CONFIG_DATA['config']`